build.sc

```diff
@@ -51,8 +51,7 @@ object Deps {
   val coursier = ivy"io.get-coursier::coursier:2.0.9"
   val flywayCore = ivy"org.flywaydb:flyway-core:6.5.7"
   val graphvizJava = ivy"guru.nidi:graphviz-java:0.18.0"
-  // Warning: Avoid ipcsocket version 1.3.0, as it caused many failures on CI
-  val ipcsocket = ivy"org.scala-sbt.ipcsocket:ipcsocket:1.0.1"
+  val ipcsocket = ivy"org.scala-sbt.ipcsocket:ipcsocket:1.3.0"
```

Comment:
Is the version bump required for making the exception visible? If not please avoid this bump for now, as we had some trouble with version 1.3.0 in the past.